lib/datasource/index.ts

```diff
@@ -124,3 +126,51 @@ export function getDigest(
     value
   );
 }
+export function sanitizeSourceUrl(rawSourceUrl) {
+  let repoUrlTmp: string;
+  if (!rawSourceUrl.endsWith('.git')) {
+    logger.debug(`${rawSourceUrl} is not a valid git url.`);
```

Comment:
Are you sure this is a valid assumption? I would prefer to use `hostedGitInfo` as the first step anyway

Reply:
hostedGitInfo requires that the uri end in git, otherwise it just returns undefined.

Reply:
That's not the impression that I got from their docs:

![image](https://user-images.githubusercontent.com/6311784/67270538-277ad800-f4b9-11e9-94cb-0e39765bc0a5.png)

Reply:
Ah, my mistake, I got a few errors while testing it via a script I wrote to test my custom function.
Will fix.

Reply:
Fixed.